diff --git a/node_modules/app-builder-lib/out/targets/nsis/NsisTarget.js b/node_modules/app-builder-lib/out/targets/nsis/NsisTarget.js
index 92cac56..bc27968 100644
--- a/node_modules/app-builder-lib/out/targets/nsis/NsisTarget.js
+++ b/node_modules/app-builder-lib/out/targets/nsis/NsisTarget.js
@@ -353,7 +353,7 @@ class NsisTarget extends core_1.Target {
         await this.executeMakensis(defines, commands, sharedHeader + (await this.computeFinalScript(script, false, archs)));
         // http://forums.winamp.com/showthread.php?p=3078545
         // TODO: remove workaround when wine is fully upgraded to 11
-        if ((0, macosVersion_1.isMacOsCatalina)()) {
+        if ((0, macosVersion_1.isMacOsCatalina)() || process.env.ARIADNE_NSIS_NO_WINE === "true") {
             try {
                 await nsisUtil_1.UninstallerReader.exec(installerPath, uninstallerPath);
             }
diff --git a/node_modules/app-builder-lib/out/targets/nsis/nsisUtil.js b/node_modules/app-builder-lib/out/targets/nsis/nsisUtil.js
index 3f2ca1f..c847c14 100644
--- a/node_modules/app-builder-lib/out/targets/nsis/nsisUtil.js
+++ b/node_modules/app-builder-lib/out/targets/nsis/nsisUtil.js
@@ -175,7 +175,7 @@ class UninstallerReader {
                 }
                 default: {
                     if (rawPointer !== 0 && rawSize !== 0) {
-                        throw new Error("Unsupported section '" + name + "'.");
+                        nsisOffset = Math.max(rawPointer + rawSize, nsisOffset);
                     }
                     break;
                 }
